armory/data/datasets.py

```diff
@@ -21,11 +21,29 @@
 from art.data_generators import DataGenerator
 from armory.data.utils import curl, download_file_from_s3
 from armory import paths
+from armory.data.resisc45 import resisc45_split  # noqa: F401
+
 
 os.environ["KMP_WARNINGS"] = "0"
 
 logger = logging.getLogger(__name__)
 
+CHECKSUMS_DIR = os.path.join(os.path.dirname(__file__), "url_checksums")
+tfds.download.add_checksums_dir(CHECKSUMS_DIR)
+
+
+def use_host_paths():
+    global DEFAULT_DATASET_DIR
+    DEFAULT_DATASET_DIR = paths.host().dataset_dir
+
+
+def use_docker_paths():
+    global DEFAULT_DATASET_DIR
+    DEFAULT_DATASET_DIR = paths.docker().dataset_dir
+
+
+use_docker_paths()
```

Comment:
Not following why these are needed? `use_host_paths()` is not a configurable option so can we remove these globals/functions and just call the paths.docker() which already caches the values so it won't be loaded many times?

Reply:
Removed.